warehouse/sponsors/__init__.py

```diff
@@ -22,3 +24,8 @@ def _sponsors(request):
 def includeme(config):
     # Add a request method which will allow to list sponsors
     config.add_request_method(_sponsors, name="sponsors", reify=True)
+
+    # Add a periodic task to update sponsors table
+    config.add_periodic_task(
+        crontab(minute=0, hour=3, day_of_week=0), update_pypi_sponsors
```

Comment:
FYI, this is 3AM UTC. Not sure if that was the original intention (also not sure if it really matters!)